tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/OpenTelemetrySdkTests.cs

```diff
@@ -158,7 +158,7 @@ public void IntegrationDisabled(string packageVersion)
             using (var agent = EnvironmentHelper.GetMockAgent())
             using (RunSampleAndWaitForExit(agent, packageVersion: packageVersion))
             {
-                var spans = agent.WaitForSpans(1, 2000);
+                var spans = agent.Spans;
```

Comment:
Andrew mentioned previously that when we expect 0 spans that instead of doing a `WaitForSpans` we can do a `agent.Spans` which seems to work

Reply:
To give a bit more context, by the time this code runs, the process has already exited. So it's _very_ unlikely we're going to get any more spans coming through - that could only happen if the process is killed literally while we're sending spans to the agent, which really shouldn't happen anyway. So the delay is probably _technically_ more thorough, but really not necessary :+1: